snippets/terms.md

```diff
@@ -46,3 +46,6 @@ We have written [detailed documentation](https://docs.opensafely.org/) about the
 We have a [Q&A forum](https://github.com/opensafely/documentation/discussions).
 
 We also offer researchers a [co-pilot](https://www.bennett.ox.ac.uk/blog/2021/08/opensafely-co-pilot-programme-assisting-users-on-their-opensafely-journey/): a member of the OpenSAFELY team who joins you on your OpenSAFELY journey, to help you achieve a completed analysis within an agreed timeframe (usually 4 weeks).
+
+## Before you start your application
+Please download our PDF application here, as we've added a checklist at the end to help you before you finally submit your application online
```

Comment:
Remember to replace `url` with the URL of the PDF application.

```suggestion
Please download our PDF application [here](url), as we've added a checklist at the end to help you before you finally submit your application online
```

Reply:
Yes, that was what I was hoping for with this, but I am unable to code this just yet with my current skill set. Sorry.

Reply:
Perhaps @ghickman could help. George, is there a place on job-server for uploads?

Reply:
Not really, and I don't want to tie the IG team to committing to job-server either.

@LiamHart-hub – can you host the file on Google Drive and use the link from there?

Reply:
Unfortunately, we have closed access to Google Drive docs to just Datalab employees only.

Reply:
@ghickman Are there any other possibilities that we could try?

Reply:
Are we able to special case single documents on GDrive?

While we can host files in job-server, it means that any updates to the file require a member of the pipeline team to make the change, which would have to be balanced against the priorities we've been set.  I always prefer colleagues being able to self serve where possible!

Reply:
@ghickman hopefully this link should work as a solution https://drive.google.com/file/d/1f6PzT-wqR2FExrQ2FSwZSV8XNPE_p_HN/view?usp=sharing 
Let me know if it doesn't and I will come up with another alternative

Reply:
@LiamHart-hub – that works when I'm not logged into a Google account 👍  Are you happy to update your changes with Iain's suggestions and the link you've generated?

Reply:
Yes, please.

Reply:
@LiamHart-hub – can I just confirm that _you_ are happy to make the changes and you don't need myself or Iain to pick them up?

Reply:
Sorry, if I have caused any confusion, to confirm I am happy to make these changes and have updated the description above which outline your suggestions - 

"Adding a new heading and description that will allow potential researchers to access a link to a copy of our application form and checklist before they start.
Hyperlink to application form and checklist to be added on the word 'here'
[Application form + Checklist.pdf] https://drive.google.com/file/d/1f6PzT-wqR2FExrQ2FSwZSV8XNPE_p_HN/view"

Reply:
You still need to actually edit the page, Liam.

Reply:
Sorry, I think that is now changed @iaindillingham